src/Utilities/TagsModal.test.js

```diff
@@ -26,15 +26,39 @@ describe('TagsModal', () => {
   });
   describe('DOM', () => {
     it('should render loading state correctly', () => {
-      const store = mockStore({});
-      const wrapper = mount(
+      const store = mockStore({
+        entities: {
+          showTagDialog: true,
+          tagModalLoaded: false,
+          activeSystemTag: {
+            tagsLoaded: false,
+          },
+        },
+      });
+      const view = render(
         <Provider store={store}>
           <TagsModal />
         </Provider>
       );
+
+      // TODO: improve skeleton accessibility
+      // baseElement is used since the PF modal is rendered outside parent div
```

Comment:
I'm a bit confused by this. Is this meant to mean to add a test for the loading state in later on?

Reply:
Hmm, I think I wanted to improve the skeleton rows attributes and add something like aria-busy=true or other aria label to mark the rows as loading. With the idea that this would also help while testing with RTL. Since the component is imported from FEC, I decided to put this off and thus left the snapshot check.

Reply:
I know see that the <table> has aria-label="Loading" attribute. I will replace the snapshot check with just asserting this attribute, but also leave the note so that we can also query rows (let's say, "expect 15 skeleton rows" rendered) in the future.